JASP-Engine/JASP/R/principalcomponentanalysis.R

```diff
@@ -352,36 +440,29 @@ PrincipalComponentAnalysis <- function(dataset = NULL, options, perform = "run",
       }
       
     }
-    
     names( Loadings[["data"]][[i]]) <- sapply(Loadings[["schema"]][["fields"]],"[[",'name')
   }
-  
+
   return(Loadings)
 }
 
 # Init:
-.initializePCA <- function(dataset, options, perform) {
+.initialize <- function(dataset, options, perform) {
   
```

Comment:
@hummyhummy Put .initialize in a common Efa PCA file efaPcaCommon.R and be careful for namespace crash.

Reply:
changed to .initializePCAEFA, which is a common function only in file principalcomponentanalysis.R